z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -372,20 +439,7 @@ The playbook creates one Security group for the Control Plane and one for the Co
 ```sh
 $ ansible-playbook -i inventory.yaml configure-network.yaml
 ```
-
-The playbook creates a default VLAN network and a subnet. The subnet obeys `os_subnet_range`. By default, if the `os_subnet_range` is '172.26.0.0/16', the allocation pools will be '172.26.0.10-172.26.255.254'. The allocation pool can be configured by `allocation_pool_start` and `allocation_pool_end`. Finally, the playbook creates the nodes' ports. 
-
-***Note***: If your Bastion node was not VLAN network type, the bootstrap node should be connected confused with bastion node. Please use of existing network.
-
-
-**Use existing network**
-
-If you want to use an existing netowrk instead of creating a new network, you can specify the 
-network name and subnet UUID, for example:
-
-```
-$ ansible-playbook -i inventory.yaml configure-network.yaml -e os_network="<Network_name>" -e os_subnet="<Network_subnet>"
-```
+The playbook use an existing network to deploy OCP, so it will overwrite DNS nameserver of this network card, please attention other application will not be interrupted.
```

Comment:
`it will overwrite DNS nameserver of this network card` -> `it will overwrite DNS nameserver of this network`

I don't think we need add `please attention other application will not be interrupted.`, the warning about overwriting DNS namesever is enough.

Reply:
Update it.